docs/concepts/protocol/07-airstreams.mdx

```diff
@@ -0,0 +1,80 @@
+---
+id: "airstreams"
+sidebar_position: 7
+title: "Airstreams"
+---
+
+:::note
+
+Please refer to the [airdrop section](/concepts/use-cases#airdrops) of our use-cases page for an explanation regarding
+why streaming an airdrop makes sense.
+
+:::
+
+## What are Airstreams?
+
+The gist of Airstreams is that instead of airdropping the entirety of the tokens all at once, airdrop recipients receive
+a fraction of the tokens every second through a token stream.
+
+The beauty of it is that airdrop recipients are forced to think long-term and keep the project’s future as their first
+and foremost priority. They are forced to, because instead of receiving all the tokens at once, they receive them over
+time.The gorgeous Airstreams interface
+
+:::info
+
+An airstream can have a duration of a few days, weeks, months and even years. You could, for example, vest the airdrop
+of your new token over 4 years. Or 6 months. You get to choose.
+
+:::
+
+Airstreams not only create the right incentives, but also ensure that the price of your token won’t crash on day one.
```

Comment:
I think instead of saying that "it ensures that price of your token won't crash on day one" we can also consider saying that "it prevents airdrop recipients from dumping their tokens on day 1". Saying that price won't crash may sound a bit inappropriate as nothing can ensure that.

Reply:
we are already mentioning "dumping their tokens below":
"This has been the case for many airdrops, where a large percentage of the airdrop recipients dumped immediately after claiming their tokens."

> Saying that price won't crash may sound a bit inappropriate as nothing can ensure that.

agree with this, what about:
"it prevents the price fluctuation on day one"